dashboard/app/models/levels/level.rb

```diff
@@ -26,6 +26,9 @@
 class Level < ActiveRecord::Base
   belongs_to :game
   has_and_belongs_to_many :concepts
+  belongs_to :script
+  belongs_to :lesson_group
+  belongs_to :lesson
```

Comment:
remove these unused associations? `belongs_to` would normally imply that this model contains a `script_id`, `lesson_group_id`, and `lesson_id`. in this case, those fields are not present, so I'm not sure what effect these have.

if you did want to add associations in the opposite direction here, it's possible you could do 
```
has_many :lessons, through: :script_levels
has_many: lesson_groups, through: :lessons
...
```